include/xrpl/protocol/Indexes.h

```diff
@@ -287,6 +287,9 @@ did(AccountID const& account) noexcept;
 Keylet
 oracle(AccountID const& account, std::uint32_t const& documentID) noexcept;
 
+Keylet
```

Comment:
You'll need a second Keylet here to access using the sfDomainID.

Reply:
fixed